### PR TITLE
Added API Explorer app 

### PR DESCRIPTION
**Pull request by tstatler**
@lparis @yhyakuna, I know you've already done this, but it would be great if you could try to follow the steps and get it working. 

cc @jamiesmith
Fixes #ENGT-8441

### CODE REVIEW

File: api-explorer/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+![Alt text](explorer.png "Optional title")
+   
+## Providing your API token {#token}
+
+To make API calls you need to provide the API Explorer with your API token. An easy to obtain your API token is from the `$HOME/.apc` file. This file is created by APC and contains the API tokens for each cluster you have targeted.
+   
+**To provide your API token**:
+
+1. Open `~./apc` (or `$HOME/.apc`) in a text editor.
```
Comment (yhyakuna) [inline api-explorer/README.md]: Typo:  `~./apc` should be `~/.apc`

Comment (tstatler) [inline api-explorer/README.md]: Thanks, fixed.

**Comment by yhyakuna** [comment]
LGTM -- I've tested this, and verified that it works with my local web server.  Let's merge it!

**Comment by yhyakuna** [comment]
my mistake

**Comment by yhyakuna** [comment]
LGTM -- tested & verified

**Pull request merged**